Source/CesiumRuntime/Public/CesiumGlobeAnchorComponent.h

```diff
@@ -32,16 +31,64 @@ class CESIUMRUNTIME_API UCesiumGlobeAnchorComponent : public UActorComponent {
    * If this is null, the Component will find and use the first Georeference
    * Actor in the level, or create one if necessary. To get the active/effective
    * Georeference from Blueprints or C++, use ResolvedGeoreference instead.
+   *
+   * If setting this property changes the CesiumGeoreference, the globe position
+   * will be maintained and the Actor's transform will be updated according to
+   * the new CesiumGeoreference.
    */
   UPROPERTY(
       EditAnywhere,
       BlueprintReadWrite,
       BlueprintGetter = GetGeoreference,
       BlueprintSetter = SetGeoreference,
-      Category = "Cesium|Georeference",
+      Category = "Cesium",
       Meta = (AllowPrivateAccess))
   TSoftObjectPtr<ACesiumGeoreference> Georeference = nullptr;
 
+  /**
+   * Whether to adjust the Actor's orientation based on globe curvature as the
+   * Actor moves.
+   *
+   * The Earth is not flat, so as we move across its surface, the direction of
+   * "up" changes. If we ignore this fact and leave an object's orientation
+   * unchanged as it moves over the globe surface, the object will become
+   * increasingly tilted and eventually be completely upside-down when we arrive
+   * at the opposite side of the globe.
+   *
+   * When this setting is enabled, this Component will automatically apply a
+   * rotation to the Actor to account for globe curvature any time the Actor's
+   * position on the globe changes.
+   *
+   * This property should usually be enabled, but it may be useful to disable it
+   * when your application already accounts for globe curvature itself when it
+   * updates an Actor's position and orientation, because in that case the Actor
+   * would be over-rotated.
+   */
+  UPROPERTY(
+      EditAnywhere,
+      BlueprintReadWrite,
+      Category = "Cesium",
+      BlueprintGetter = GetAdjustOrientationForGlobeWhenMoving,
+      BlueprintSetter = SetAdjustOrientationForGlobeWhenMoving,
+      Category = "Cesium",
+      Meta = (AllowPrivateAccess))
+  bool AdjustOrientationForGlobeWhenMoving = true;
+
+  /**
+   * Using the teleport flag will move objects to the updated transform
+   * immediately and without affecting their velocity. This is useful when
+   * working with physics actors that maintain an internal velocity which we do
+   * not want to change when updating location.
+   */
+  UPROPERTY(
+      EditAnywhere,
+      BlueprintReadWrite,
+      BlueprintGetter = GetTeleportWhenUpdatingTransform,
+      BlueprintSetter = SetTeleportWhenUpdatingTransform,
+      Category = "Cesium",
+      Meta = (AllowPrivateAccess))
+  bool TeleportWhenUpdatingTransform = true;
```

Comment:
I wrote an issue to address this, with a bit of explanation:
https://github.com/CesiumGS/cesium-unreal/issues/1231

I think it can be post-2.0 though.